src/components/Note.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import {
+  IconButton,
+  Typography,
+  ListItemText,
+  ListItemButton,
+  ListItem,
+} from "@mui/material";
+import IosShareIcon from "@mui/icons-material/IosShare";
+import DeleteIcon from "@mui/icons-material/Delete";
+import PropTypes from "prop-types";
+
+import styles from "../../src/pages/styled";
+import shortenDescription from "../../src/utils/noteShortDescription";
+
+const Note = ({ note, displayChosenNote, shareNote, removeNote }) => {
+  return (
+    <ListItem divider>
+      <ListItemButton
+        onClick={() => {
+          displayChosenNote(note);
+        }}
+      >
+        <ListItemText
+          primary={
+            <Typography style={styles.noteTitle}>{note.title}</Typography>
+          }
+          secondary={
+            <React.Fragment>
+              <Typography
+                sx={{ display: "inline-block" }}
+                component="span"
+                style={styles.noteInList}
+                className="shortDescription"
+              >
+                {shortenDescription(note.description)}
+              </Typography>
+              {note.createdAt.substr(0, 10)}
+            </React.Fragment>
+          }
+          style={styles.noteInList}
+        />
+      </ListItemButton>
+      <IconButton
+        onClick={() => shareNote(note)}
+        color="inherit"
+        component="span"
+      >
+        <IosShareIcon style={styles.shareIcon} />
+      </IconButton>
+      <IconButton
+        onClick={() => removeNote(note.id)}

```

Comment:
There is no need to send an id, because you already have it in state near with those function